foxy/distribution.yaml

```diff
@@ -2846,6 +2846,25 @@ repositories:
       url: https://github.com/ros-controls/ros2_control.git
       version: master
     status: developed
+  ros2_controllers:
+    doc:
+      type: git
+      url: https://github.com/ros-controls/ros2_controllers.git
+      version: master
+    release:
+      packages:
+      - forward_command_controller
+      - joint_state_controller
+      - joint_trajectory_controller
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/bmagyar/ros2_controllers-release.git
```

Comment:
@bmagyar hmm, why is the release repository different from the one used for Dashing?

Reply:
because a week ago when I submitted this, I had no access to that repo. Newer releases will go into that release repo

Reply:
@bmagyar would you be willing to push these changes to the ros2-gbp repository and recreate this pull request from there to keep all the release tags in one place? There's no technical hurdle to the sharding but it makes release forensics more challenging in the future.

Reply:
done https://github.com/ros/rosdistro/pull/27872